### PR TITLE
Fixed bugs , make ROSTopic and RvizTopic able to launch

### PR DESCRIPTION
**Pull request by Corsair-cxs**
**What kind of change does this PR introduce?** (check at least one)

- [X] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch for ros_openvino_toolkit (or to a previous version branch), _not_ the `master` branch
- [ ] New/updated tests are included

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

File: vino_launch/param/pipeline_object.yaml [hunk 1]
```diff
@@ -3,19 +3,21 @@ Pipelines:
   inputs: [StandardCamera]
   infers:
     - name: ObjectDetection
-      model: /opt/openvino_toolkit/models/object_detection/mobilenet-ssd/caffe/output/FP16/mobilenet-ssd.xml
```
Comment (LewisLiuPub) [inline vino_launch/param/pipeline_object.yaml:6]: Don't change the model path to other folder, just keep the file unchanged.

File: vino_launch/launch/pipeline_object.launch [hunk 2]
```diff
@@ -8,4 +8,7 @@
         <remap from ="/openvino_toolkit/detected_objects" to = "/ros_openvino_toolkit/detected_objects" />
         <remap from ="/openvino_toolkit/images" to = "/ros_openvino_toolkit/image_rviz" />
   </node>
+  <node name="rviz" pkg="rviz" type="rviz" required="true" output="screen">
```
Comment (LewisLiuPub) [inline vino_launch/launch/pipeline_object.launch:11]: please create a new launch file for rviz launching only. Don't add rviz node here.

Comment (Corsair-cxs) [reply vino_launch/launch/pipeline_object.launch:11]: Okey, I'll refine it!

Comment (Corsair-cxs) [reply vino_launch/launch/pipeline_object.launch:11]: OK, I'll refine it!

**Comment by LewisLiuPub** [comment]
That's ok with me.

**Pull request merged**